app/src/main/java/org/mifos/mobile/ui/fragments/NewHomeFragment.kt

```diff
@@ -0,0 +1,104 @@
+package org.mifos.mobile.ui.fragments
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.core.content.ContextCompat
+import kotlinx.android.synthetic.main.content_fragment_home.*
+import kotlinx.android.synthetic.main.fragment_home.*
+import org.mifos.mobile.R
+import org.mifos.mobile.ui.activities.NotificationActivity
+import org.mifos.mobile.ui.fragments.base.BaseFragment
+
+class NewHomeFragment : BaseFragment(), View.OnClickListener {
+
+    private lateinit var rootView: View
+    private var currentSelectedButton = 1;
```

Comment:
do we need this variable at the class level, clarify? (method level should serve the purpose)

Reply:
Yes, I think we need this variable at class level as it used inside two functions `onClick` and `updateUI`.

Reply:
Firstly, `currentSelectedButton` is not needed at class-level. If you wanna use this approach, you can add an arg for `updateUI`.

Let's create separate methods, over using `switch`